prusti-encoder/src/encoders/mir_impure.rs

```diff
@@ -64,7 +68,7 @@ impl TaskEncoder for MirImpureEncoder {
         *task
     }
 
-    fn do_encode_full<'tcx: 'vir, 'vir>(
+    fn do_encode_full<'tcx: 'vir, 'vir: 'tcx>(
```

Comment:
What forced the invariance?

Reply:
I hadn't been able to figure it out, so that's why it was like this. Now I've tracked it down to having `'vir` here: https://github.com/Aurel300/prusti-dev/pull/14/files#diff-7e0bdbbcc42baa44b6008e26f05fcd4e65edf5b564fe207827b8dc8850d542f8R117